channel.go

```diff
@@ -0,0 +1,340 @@
+package cli
+
+import (
+	"encoding/json"
+	"time"
+
+	stream "github.com/GetStream/stream-chat-go/v5"
+	"github.com/cheynewallace/tabby"
+	"github.com/urfave/cli/v2"
+)
+
+func NewChannelCmd(config *Config) *cli.Command {
+	return &cli.Command{
+		Name:        "channel",
+		Usage:       "Manage channels.",
+		Description: "Manage channels.",
+		Subcommands: []*cli.Command{
+			getChannelCmd(config),
+			listChannelsCmd(config),
+			createChannelCmd(config),
+			deleteChannelCmd(config),
+			updateChannelCmd(config),
+		},
+	}
+}
+
+func getChannelCmd(config *Config) *cli.Command {
+	return &cli.Command{
+		Name:        "get",
+		Usage:       "Get a channel by channel type and channel name.",
+		UsageText:   "stream-cli channel get --type [channel-type] --name [channel-name]",
+		Description: "Get a channel by channel type and channel name.",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:     "type",
+				Aliases:  []string{"t"},
+				Usage:    "The type of the channel. Such as 'messaging' or 'livestream'.",
+				Required: true,
+			},
+			&cli.StringFlag{
+				Name:     "name",
+				Aliases:  []string{"n"},
+				Usage:    "The name of the channel.",
+				Required: true,
+			},
+			&cli.BoolFlag{
+				Name:     "json",
+				Usage:    "Print the raw JSON representation of the API response.",
+				Required: false,
+			},
+		},
+		Action: func(ctx *cli.Context) error {
+			r, err := getOrCreateChannel(ctx, config)
```

Comment:
does it make sense to do GetOrCreate? What if the user uses this to see if the channel exists or not?

Reply:
Well, @yaziine recommended me to throw an exception if a channel exists already. We should make up our minds. :)

Reply:
Imho we should follow the api/client, if that makes sense.

I guess there's no design doc for this, so everyone will dump his opinion here 😅

Reply:
I'm still a bit confused about the implementation of `getOrCreateChannel`, it does it create the channel if it doesn't exist, with user_id `"stream-go-cli"`?

Reply:
So, `user` is a mandatory parameter in create channel but not mandatory in in get channel.

But still, it throws an error if you don't provide user id, so we provide a dummy one. That id isn't being used when it's a get operation, as far as I understand.

Reply:
but getOrCreateChannel only does CreateChannel

Reply:
talking about this https://github.com/GetStream/stream-cli/pull/76/files/62afc38ddd2ce83c651af7ec049d12671d019648#diff-7b28604b0113c10108ba3d8fc11fe87060c0e3bdeb770d9dfb0d3f4f9daa6edbR145

Reply:
damn, you're right 🤦

Reply:
fixed